standalone/extHistoryEnumerator.py

```diff
@@ -0,0 +1,66 @@
+"""
+This script enumerates all occurrences of a text, based on a regex.
+The following search example returns all soapaction names found in 
+    the history, be it in the header or body.
+"""
+
+from org.parosproxy.paros.control import Control;
+from org.parosproxy.paros.extension.history import ExtensionHistory;
+from org.parosproxy.paros.model import Model;
+from org.parosproxy.paros.model import HistoryReference;
+import re;
+from org.parosproxy.paros.view import AbstractFrame;
+from javax.swing import JPanel;
+from org.zaproxy.zap.utils import ZapTextArea;
+
+sessionId = Model.getSingleton().getSession();
+tbHist = Model.getSingleton().getDb().getTableHistory();
+regex='soapaction';
```

Comment:
Could you add a comment to this line so that it stands out to other users that this is the variable value they need to modify to their own needs. (You could also quote the variable name in the lead-in block comment.) 

Would it be worth making the example be an actual regex? (`'soapactio.*'` or whatever?)